cmake/kconfig.cmake

```diff
@@ -26,6 +26,8 @@ if(OVERLAY_CONFIG)
   string(REPLACE " " ";" OVERLAY_CONFIG_AS_LIST "${OVERLAY_CONFIG}")
 endif()
 
+string(REPLACE ";" "?" DTS_ROOT_BINDINGS "${DTS_ROOT_BINDINGS}")
```

Comment:
Would be nice to have a comment above it (that explains quickly how CMake handles lists too, maybe, because that was confusing to me).

Reply:
Done